service/src/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -119,6 +122,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.collect.EvictingQueue;
```

Comment:
Nit: why new line before this?

Reply:
@pvary These import changes/spaces/orders are checkstyle rules:

https://checkstyle.sourceforge.io/config_imports.html#ImportOrder

```none
adds a separation between type import groups : ensures that a blank line sit between each group
```